freqtrade/tests/test_arguments.py

```diff
@@ -196,17 +195,16 @@ def test_plot_dataframe_options() -> None:
         '--plot-limit', '30',
         '-p', 'UNITTEST/BTC',
     ]
-    arguments = Arguments(args, '')
-    arguments.build_args(ARGS_PLOT_DATAFRAME)
-    pargs = arguments.parse_args(True)
+    arguments = Arguments(args, '', no_default_config=True)  ## Why True?
```

Comment:
honestly - i think for legacy reasons (there's no reason i can see to do this for plot scripts ...)

Reply:
removed it (rebased)